pkg/test/fakes/policy/backend.go

```diff
@@ -75,7 +75,7 @@ func (b *Backend) Port() int {
 
 // Start the gRPC service for the policy backend.
 func (b *Backend) Start() error {
-	scope.Infof("Starting Policy Backend at port: %d", b.port)
```

Comment:
Any reason for removing the port number? It could be helpful in debugging.

Reply:
When starting this is 0 and actually creates confusion :(
It gets allocated later and thus I added it in the log below:
scope.Infof("Starting the GRPC service at port: %d", b.port)